NetworkPkg/Ip6Dxe/Ip6Nd.c

```diff
@@ -1416,7 +1458,14 @@ Ip6SendNeighborSolicit (
                                                  sizeof (IP6_ETHER_ADDR_OPTION),
                                                  FALSE
                                                  );
-    ASSERT (LinkLayerOption != NULL);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (LinkLayerOption == NULL) {

```

Comment:
`NetBufFree (IcmpHead);` in this block.

Reply:
These `NetbufAllocSpace()` calls appear to be carving out space from an already allocated buffer so I'll drop my comment on them.